tests/integration_tests/data_steward/validation/ehr_union_test.py

```diff
@@ -77,12 +79,14 @@ def _create_hpo_table(self, hpo_id, table, dataset_id):
     def _load_datasets(self):
         """
         Load five persons data for each test hpo
-        """
         # expected_tables is for testing output
         # it maps table name to list of expected records ex: "unioned_ehr_visit_occurrence" -> [{}, {}, ...]
+        :param pto: Flag indicating whether load is for person_to_observation tests
+        """
         expected_tables = dict()
         running_jobs = []
-        for cdm_table in resources.CDM_TABLES:
+        cdm_tables = resources.CDM_TABLES
```

Comment:
Is there a particular reason that `cdm_tables` was created?

Reply:
I had created it to mock `resources.CDM_TABLES` but later realized that was not necessary since I can mock `tables_to_map`. I'll remove it